### PR TITLE
Switch to non-internal dep.

### PR DESCRIPTION
**Pull request by dt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+Fork of `rsc.io/benchstat` that adds json output.
```
Comment (tamird) [inline README.md:1]: it doesn't actually, right? not yet, at least.

**Comment by tamird** [comment]
I think you mean rsc's rather than rss. also this commit doesn't fork it, so just remove all that.

File: README.md (hunk 1 above)
Comment (dt) [inline README.md:1]: https://github.com/cockroachdb/benchstat/pull/3

**Comment by tamird** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (tamird) [inline README.md:1]: :twisted_rightwards_arrows:

**Comment by dt** [comment]
Open question: should this fork be `roachbenchstat` or something to avoid a name collision in PATH?
Seems like it could get confusing if flags work for one person running `benchstat` and not another, vs a more obvious "not found" error?

**Comment by tamird** [comment]
glock should take care of having everyone on the same binary.

**Comment by petermattis** [comment]
I think we should leave the name as `benchstat`. Just as we left the name `stress` for  https://github.com/cockroachdb/stress.

**Comment by bdarnell** [comment]
FWIW, I'd be in favor of changing the names of both this tool and our forked `stress`. `glock` makes sure that you have our version installed somewhere but it doesn't guarantee that it's first on the `$PATH` (my current setup doesn't modify `$PATH` when it switches `$GOPATH` so I run into issues with this occasionally)

**Comment by dt** [comment]
=ben

When we're relying on a difference in behavior, having a different name makes the failure mode easier to recognize -- a "not found" message points instantly to the issue whereas as an incompatible flag, or worse a slight change in output format, could be much harder to spot (or could even manifest as a some downstream tool choking on malformed input, far away from the actual problem).

**Pull request merged**